references/classification/presets.py

```diff
@@ -69,11 +69,10 @@ def __init__(
         backend="pil",
     ):
         trans = []
-
         backend = backend.lower()
         if backend == "tensor":
```

Comment:
Probably this should change to 
```suggestion
        if backend == "tensor" or backend == "pil":
```